.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: npm install
```

Comment:
Maybe we can consider adding .gitpod.yml to .gitignore?

Reply:
File added to .gitignore , but then also thought that users of the package dont need to have this file, sine it is only relevant for users of [gitpo.io](https://gitpod.io/) which is an online IDE , to file has been deleted in the latest commit